feature-libs/product-configurator/rulebased/components/attribute/types/radio-button/configurator-attribute-radio-button.component.ts

```diff
@@ -20,6 +25,8 @@ export class ConfiguratorAttributeRadioButtonComponent
   extends ConfiguratorAttributeSingleSelectionBaseComponent
   implements OnInit
 {
+  @Input() uiKeyPrefix: string = '';
```

Comment:
`@Input() uiKeyPrefix: string = '';` should be moved to `ConfiguratorAttributeSingleSelectionBaseComponent`

Reply:
not all implemntaions of ConfiguratorAttributeSingleSelectionBaseComponent use the uiKeyPrefix